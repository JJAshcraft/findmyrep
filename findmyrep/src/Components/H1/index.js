import styled from "styled-components";

const H1 = styled.h1`
  font-size: 20px;
  font-weight: 500;
  color: #06a7ed;
`;

export default H1;
