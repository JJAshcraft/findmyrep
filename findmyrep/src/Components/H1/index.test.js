import React from "react";
import renderer from "react-test-renderer";
import "jest-styled-components";
import H1 from "./index.js";

test("H1 component renders with proper styling", () => {
  const tree = renderer.create(<H1 />).toJSON();
  expect(tree).toMatchSnapshot();
  expect(tree).toHaveStyleRule("color", "#06a7ed");
  expect(tree).toHaveStyleRule("font-size", "20px");
  expect(tree).toHaveStyleRule("font-weight", "500");
});
