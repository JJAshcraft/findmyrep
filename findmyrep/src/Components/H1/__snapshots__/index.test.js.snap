// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`H1 component renders with proper styling 1`] = `
.c0 {
  font-size: 20px;
  font-weight: 500;
  color: #06a7ed;
}

<h1
  className="c0"
/>
`;
